### ISSUE TITLE
[BUG] Cleanup not waiting for resources to remove

### ISSUE THREAD

**Issue by martin-mat** [cnti-testcatalog/testsuite#2194]
Cleanup does not wait for removing reources. This is inconsistent with how setup works, waiting for resources to be created.

**Comment by collivier** [comment]
Agree. I did add a 60s sleep between to 2 cnf_testsuite calls (via Functest Kubernetes)
